tests/testcases/040_check-network-adv.yml

```diff
@@ -14,6 +14,17 @@
     netchecker_port: 31081
 
   tasks:
+    # TODO remove debug kube_network_plugin_flo
+    - debug: var=kube_network_plugin
+
+    - name:  Fix missing route (see https://github.com/coreos/flannel/issues/1243)
```

Comment:
it is very ugly hack (((

Reply:
Yes I agree, that was mainly to validate tests are now ok with this route; do you have any idea on how to do it cleaner ? or do we need to wait for a k8s/flannel fix ?

Reply:
`ethtool --offload flannel.1 rx off tx off` 
looks more correct fix